### PR TITLE
samples: nrf9160: gps: Improve GPS sample output

### PR DESCRIPTION
**Pull request by tokangas**
Improvements for GPS sample output, for example NMEA strings are now
printed all the time, not just when the GPS fix is valid. Adds also
support for NMEA only mode.

Signed-off-by: Tommi Kangas <tommi.kangas@nordicsemi.no>

### CODE REVIEW

**Comment by tokangas** [comment]
Added all changes suggested by @umapraseeda

File: samples/nrf9160/gps/Kconfig [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "GPS sample"
+
+config GPS_SAMPLE_NMEA_ONLY
+	bool "Output only NMEA strings"
+	default n
```
Comment (rlubos) [inline samples/nrf9160/gps/Kconfig:11]: nit: `default n` is unnecessary.

File: samples/nrf9160/gps/Kconfig [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	help
+		Outputs only NMEA strings from the GPS.
```
Comment (jtguggedal) [inline samples/nrf9160/gps/Kconfig:13]: Nit: indentation for help text is 2 spaces (for unknown reasons)
```
        help
          Outputs only NMEA strings from the GPS.
```

File: samples/nrf9160/gps/src/main.c [hunk 3]
```diff
@@ -48,10 +48,9 @@ static int            gnss_fd;
 static char           nmea_strings[10][NRF_GNSS_NMEA_MAX_LEN];
 static u32_t          nmea_string_cnt;
 
-static bool           got_first_fix;
-static bool           update_terminal;
+static bool           got_fix;
 static u64_t          fix_timestamp;
-nrf_gnss_data_frame_t last_fix;
+nrf_gnss_data_frame_t last_pvt;
```
Comment (jtguggedal) [inline samples/nrf9160/gps/src/main.c:53]: Looks like this variable could be `static`

File: samples/nrf9160/gps/src/supl_support.c [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019 Nordic Semiconductor ASA
+ * Copyright (c) 2020 Nordic Semiconductor ASA
```
Comment (jtguggedal) [inline samples/nrf9160/gps/src/supl_support.c:2]: This one should probably stay as 2019

**Comment by tokangas** [comment]
Thanks, I made all suggested changes. Some hassle with the indentations, but it should be ok now. :)

**Comment by rlubos** [comment]
@tokangas This PR looks good to go, but the CI is blocking it, could we get a rebase here?

**Comment by tokangas** [comment]
@rlubos Rebased.

**Pull request merged**